types/react-native/index.d.ts

```diff
@@ -2938,6 +2938,16 @@ export interface RecyclerViewBackedScrollViewStatic extends ScrollResponderMixin
     ): void;
 
     /**
+     * A helper function that scrolls to the end of the scrollview;
+     * If this is a vertical ScrollView, it scrolls to the bottom.
+     * If this is a horizontal ScrollView scrolls to the right.
+     *
+     * The options object has an animated prop, that enables the scrolling animation or not.
+     * The animated prop defaults to true
+     */
+    scrollToEnd(options?: {animated: boolean}): void;
+
+    /**
```

Comment:
@iRoachie According to http://twitter.com/roach_iam/status/854335766222262273 you intended to add this to `ScrollViewStatic`, but this PR adds it to `RecyclerViewBackedScrollViewStatic` instead. Can you make a new PR that moves it to the right place?